packages/synapse-react-client/src/components/QueryContext/QueryContext.tsx

```diff
@@ -198,4 +200,10 @@ export function useInfiniteQueryContext(): InfiniteQueryContextType {
   return context as InfiniteQueryContextType
 }
 
+export type CombineRangeFacetConfig = {
+  minFacetColumn: string
+  maxFacetColumn: string
+  label: string
+}
+
```

Comment:
Do you think it makes more sense to have this here in `QueryContext`/`QueryWrapper` or `QueryVisualizationContext`/`QueryVisualizationWrapper`? I'm starting to think that this is probably not the best way to separate responsibilities, since many UI config tweaks may actually need to modify the query itself.